### PR TITLE
Add HMPID to AO2D conversion

### PR DESCRIPTION
**Pull request by njacazio**
Hi @gvolpe79 this is the changes that we would port to the O2 data format, do we need anything else?

### CODE REVIEW

**Comment by pzhristov** [comment]
+1

**Comment by ktf** [comment]
Please remove the draft if you are ready.

**Comment by ktf** [comment]
Changing to non draft, since there was no reply.

**Comment by njacazio** [comment]
Hi @ktf, I did not reply because I would prefer not to be merged yet as I would like Giacomo to validates it first

**Comment by ktf** [comment]
ok, i closed it to avoid interference with the builds while we understand the permissions race condition.

**Comment by njacazio** [comment]
@ktf shall we reopen this one or is it better if we open a new one?

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx [hunk 1]
```diff
@@ -593,6 +593,7 @@ void AliAnalysisTaskAO2Dconverter::InitTF(ULong64_t tfId)
     tTracks->Branch("fTPCSignal", &tracks.fTPCSignal, "fTPCSignal/F");
     tTracks->Branch("fTRDSignal", &tracks.fTRDSignal, "fTRDSignal/F");
     tTracks->Branch("fTOFSignal", &tracks.fTOFSignal, "fTOFSignal/F");
+    tTracks->Branch("fHMPIDSignal", &tracks.fHMPIDSignal, "fHMPIDSignal/F");
```
Comment (njacazio) [inline RUN3/AliAnalysisTaskAO2Dconverter.cxx:596]: @gvolpe79 do you think we miss anything else apart from the HMPID signal?

**Comment by gvolpe79** [comment]
The HMPID signal (Cherenkov angle) is the only parameter needed for the HMPID PID! What it is not clear to me it is if the track quality cuts to select good tracks are applied at the AOD creation or at analysis level! if the latter case, we should add other information.

**Comment by njacazio** [comment]
So we need to add the mip charge, the distance between the extrapolated track and the matched cluster. 
The mip charge is maximum at 4000 (a short would be enough), this is used to cut, tracks with q_mip > 120 are accepted.
The distance would be a float to be used to cut tracks with distance > 5 cm.
I will investigate now if this has to go in a separate table

**Pull request closed without merging**